encoding/encodeInto.any.js

```diff
@@ -127,11 +127,14 @@
  Uint16Array,
  Uint32Array,
  Uint8ClampedArray,
+ typeof BigInt64Array === "function" ? BigInt64Array : undefined,
+ typeof BigUint64Array === "function" ? BigUint64Array : undefined,
```

Comment:
The `typeof` operator is needed to not cause a `ReferenceError` to be thrown while setting up the tests.

Reply:
This is exposed in all browsers. Can you give some more detail as to where this goes wrong?

Reply:
`BigInt64Array` and `BigUint64Array` are not supported in **Safari 14**, only in **Safari Technical Preview**.

Reply:
Honestly I think that's fine for these tests. That's also what all the tooling uses.

Reply:
That would lead to a **Harness status** of `FAIL`, and any tests defined after these lines to be `MISSING`.

Reply:
But only in Safari 14, right? And all the bots use Safari TP? I feel like I'm missing something.

Reply:
**Apple** likely needs to run these tests on **Safari 14** while it’s still a supported version.

Reply:
I guess it's reasonable to work around that, but the amount of tests should not end up varying as it does now. In that case just quote them all and use `new globalThis[viewAsString](...)` or some such.